index.js

```diff
@@ -56,4 +56,10 @@ verbs.forEach(function(method) {
   };
 });
 
-module.exports = MockAdapter;
+if (typeof define === 'function' && define.amd) {
+  define(MockAdapter);
+} else if (typeof exports === 'object') {
+  module.exports = MockAdapter;
+} else {
+  window.AxiosMockAdapter = MockAdapter;
```

Comment:
One problem is that not only `AxiosMockAdapter`, but also `MockAdapter` and any other variable or function in this file would get defined in the global scope. Wrapping the file contents in
```js
(function(global) {
  // ...
})(this);
```
would solve it, but I was thinking that since anyway I'd like to split up this one file into smaller files, it would perhaps be better to use webpack to create a UMD build.